.postman/api_84fc90ca-3153-4865-8b49-b91218e5d574

```diff
@@ -17,3 +17,4 @@ files[] = {"id":"23674746-92761441-6d0c-4fc6-aa04-73eccf6afd4c","path":"Interled
 rootDirectory = postman/schemas
 
 [config.relations.apiDefinition.metaData]
+type = openapi:3
```

Comment:
should this have changed?

Reply:
Postman did that. I don't know what that is. It said it changed an internal file when I clicked on the diff thing within postman